isso/js/app/templates/comment.js

```diff
@@ -7,15 +7,16 @@ var html = function (globals) {
   var svg = globals.svg;
 
   var author = comment.author ? comment.author : i18n('comment-anonymous');
+  var page_author_class = (conf["page-author-hash"]).indexOf(comment.hash) > -1 ? " isso-page-author" : "";
```

Comment:
`isso-is-page-author` instead? (implying that it's kind of a boolean to CSS editors)